@xen-orchestra/web/src/components/vm/VmManagment.vue

```diff
@@ -0,0 +1,65 @@
+<template>
+  <UiPanel class="vm-resource">
```

Comment:
```suggestion
  <UiPanel class="vm-management">
```

You can change the name in the component too